x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/response/schemas.ts

```diff
@@ -45,6 +51,13 @@ export const output_index = t.string;
 export const saved_id = t.string;
 export const timeline_id = t.string;
 export const timeline_title = t.string;
+export const throttle = t.keyof({
+  no_actions: null,
+  rule: null,
+  '1h': null,
+  '1d': null,
+  '7d': null,
+});
```

Comment:
Same comments as before, this should be just a string or a null here and not have extra keys such as `no_actions` and `rule`